### ISSUE TITLE
JAWS does not announce description as associated with aria-describedby for a fieldset with text inputs

### ISSUE THREAD

**Issue by alex-ju** [FreedomScientific/VFO-standards-support#80]
### Summary

> JAWS does not announce description as associated with `aria-describedby` for a fieldset that contains text inputs.
> 
> Example:
> 1. Go to https://codepen.io/alex-ju/full/KRQMvr/
> 2. Tab to an input inside the fieldset (i.e. the _Street address_ input)

### Expected result

> We'd expect it to announce the description (basically to behave the same it does with a fieldset that contains radios or checkboxes - see example below)
> 1. Go to https://codepen.io/alex-ju/full/wjyrVe/
> 2. Tab to an input inside the fieldset (i.e. the _Yes_ input)

### Actual result

> It doesn't announce the description.

### Example
> https://codepen.io/alex-ju/full/wjyrVe/

### Additional Information

> If `role="group"` is added to the fieldset, the description is being announced, but also the legend is being repeated which becomes too verbose.
> https://codepen.io/alex-ju/full/PeQOOO/
>
> We used this approach as [a workaround in our code](https://github.com/alphagov/govuk-frontend/pull/681).

#### JAWS version and build number
> JAWS 18.0.4534 and 17.0.2729
 
#### Operating System and version
> Windows 10 Pro

#### Browser and version:
> Internet Explorer (IE) 11.371

**Comment by mgifford** [comment]
I think we just ran into that issue here https://www.drupal.org/project/drupal/issues/3013489#comment-12855984

**Comment by jducrot** [comment]
Confirmed on IE11 + JAWS 20

**Comment by stevefaulkner** [comment]
@BenKeyFSI please review and file on internal bugzilla, once done provide bug # and title here then close the issue.

**Comment by BenKeyFSI** [comment]
I have confirmed the original issue reported by @alex-ju. However, in JAWS 20 I am seeing slightly different results in the role="group" scenario. Alex reported that setting the role attribute to group for the fieldset causes the legend to be repeated when the Postcode field gains focus.

This is not happening in JAWS 2020.

**Comment by BenKeyFSI** [comment]
Reported this issue as Bug 114342 {JAWS does not announce description as associated with aria-describedby for a fieldset with text inputs} in BugZilla.
